unidist/core/backends/mpi/core/serialization.py

```diff
@@ -9,7 +9,12 @@
 
 # Serialization libraries
 if sys.version_info[1] < 8:  # check the minor Python version
-    import pickle5 as pkl
+    try:
+        import pickle5 as pkl
+    except ImportError:
+        raise ImportError(
+            "Missing dependency 'pickle5'. Use pip or conda to install it."
+        ) from None
```

Comment:
What's need of this?

Reply:
It is used to suppress earlier context of an arisen exception to not confuse a user why we catch the exception and then throw it. Compare these two examples:
```python
def foo():
   try:
       import f
   except ImportError:
       raise ImportError("Error message")

def foo2():
   try:
       import f
   except ImportError:
       raise ImportError("Error message") from None

foo()
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-1-68aa7f47062e> in foo()
      2     try:
----> 3         import f
      4     except ImportError:

ModuleNotFoundError: No module named 'f'

During handling of the above exception, another exception occurred:

ImportError                               Traceback (most recent call last)
<ipython-input-3-c19b6d9633cf> in <module>
----> 1 foo()

<ipython-input-1-68aa7f47062e> in foo()
      3         import f
      4     except ImportError:
----> 5         raise ImportError("Error message")
      6

ImportError: Error message

foo2()
ImportError                               Traceback (most recent call last)
<ipython-input-4-33ee749b93dd> in <module>
----> 1 foo2()

<ipython-input-2-4c2204bd9dad> in foo2()
      3         import f
      4     except ImportError:
----> 5         raise ImportError("Error message") from None
      6
      7

ImportError: Error message
```

Reply:
Ok, thanks for the explanation.